templates/build-ports.p2

```diff
@@ -20,6 +22,10 @@ _build_port()
 
 for jail in ${JAIL_NAMES}
 do
+  if [ "$1" == "-y" ]; then
+      _build_port $jail
```

Comment:
I haven't tried it yet, so one question: when you skip `options` step, it means all packages are build using their default options, right?